src/web/browser/App.tsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Nav } from '@frontend/web/components/Nav/Nav';
+import { EditionDropdown } from '@frontend/web/components/EditionDropdown';
+import { MostViewedFooter } from '@frontend/web/components/MostViewed/MostViewedFooter/MostViewedFooter';
+import { MostViewedRightWrapper } from '@frontend/web/components/MostViewed/MostViewedRight/MostViewedRightWrapper';
+import { Counts } from '@frontend/web/components/Counts';
+import { RichLinkComponent } from '@frontend/web/components/elements/RichLinkComponent';
+import { ReaderRevenueLinks } from '@frontend/web/components/ReaderRevenueLinks';
+import { CMP } from '@frontend/web/components/CMP';
+import { Onwards } from '@frontend/web/components/Onwards/Onwards';
+import { SlotBodyEnd } from '@frontend/web/components/SlotBodyEnd';
+import { SubNav } from '@frontend/web/components/SubNav/SubNav';
+
+type Props = { CAPI: CAPIType; NAV: NavType };
+
+type RootType =
```

Comment:
I presume the goal here is to keep track of all the root elements we define in the DOM server-side and making sure we don't accidentally try to attach a component on the client to a root that doesn't exist. Any other reason? Definitely not preferable to just accept as a root ID to avoid having to update this type each time we add a new one?

Reply:
Honestly, I added this type to make my life easier when writing all the components inside `App`. Having it typed meant VS Code gave me a list to pick from when choosing the root property so I didn't need to switch files and copy paste!

But yeah, it's also nice for future devs to understand things. ❤️ Typescript